Essentials/src/com/earth2me/essentials/EssentialsPlayerListener.java

```diff
@@ -80,7 +80,14 @@ public void onPlayerChat(final AsyncPlayerChatEvent event) {
         final User user = ess.getUser(event.getPlayer());
         if (user.isMuted()) {
             event.setCancelled(true);
-            user.sendMessage(tl("voiceSilenced"));
+
+            if (user.getMuteReason ().equals ("")) {
```

Comment:
The space between `getMuteReason` and its brackets doesn't match the style conventions followed by the rest of EssentialsX's code.